### PR TITLE
drawpile,drawpile-server-headless: 2.1.17 -> 2.1.19

### PR DESCRIPTION
**Pull request by fgaz**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [x] (Package updates) Added a release notes entry if the change is major or breaking
  - [x] (Module updates) Added a release notes entry if the change is significant
  - [x] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/graphics/drawpile/default.nix [hunk 1]
```diff
@@ -69,23 +69,15 @@ let
...
     owner = "drawpile";
     repo = "drawpile";
     rev = version;
-    sha256 = "sha256-AFFY+FcY9ExAur13OoWR9285RZtBe6jnRIrwi5raiCM=";
+    sha256 = "sha256-MNmzcqTHfMms6q3ZilrChE5WoGzGxnAOkB0a75udA1I=";
   };
 
-  patches = [
-    # fix for libmicrohttpd 0.9.71
-    (fetchpatch {
```
Comment (SuperSandro2000) [inline pkgs/applications/graphics/drawpile/default.nix:83]: Please remove the unused fetchpatch from inputs.

Comment (veprbl) [reply pkgs/applications/graphics/drawpile/default.nix:83]: Done
9c0644262fc9b5a7c52c25b74d919c324fb34b6a

Comment (fgaz) [reply pkgs/applications/graphics/drawpile/default.nix:83]: Thanks! By the way, do we have a linter I can use to check for this kind of easily forgettable stuff?

Comment (SuperSandro2000) [reply pkgs/applications/graphics/drawpile/default.nix:83]: https://github.com/jtojnar/nixpkgs-hammering

**Pull request merged**